webapp/locales/en.json

```diff
@@ -269,11 +269,9 @@
       "name": "Download Data"
     },
     "deleteUserAccount": {
-      "name": "Delete data",
-      "contributionsCount": "Delete my {count} posts",
-      "commentedCount": "Delete my {count} comments",
```

Comment:
Are you sure that `contributionsCount` and `commentedCount` are not needed anymore?

E.g. when I run:
```sh
git grep 'deleteUserAccount.commentedCount'
```

I see one search result in `webapp/components/DeleteData/DeleteData.vue`

Reply:
Yes, I wrote another Pull Request 
Update DeleteData.vue #2447